multichain-testing/config.fusdc.yaml

```diff
@@ -40,7 +40,7 @@ chains:
         staking:
           params:
             unbonding_time: '2m'
-        interchain_accounts:
+        interchainaccounts:
```

Comment:
Nice catch here in our pairing earlier. Please make this change in all `config*.yaml` files - i see 5 more instances that need to be replaced

Reply:
Done: https://github.com/Agoric/agoric-sdk/pull/11292/commits/f895f7ec7fd7c4bfb085d30a08e1cc18c2f7250c